src/app/(proper_react)/redesign/(public)/page.tsx

```diff
@@ -14,11 +18,18 @@ export default async function Page() {
   const countryCode = getCountryCode(headers());
   const eligibleForPremium = isEligibleForPremium(countryCode, enabledFlags);
 
+  // request the profile stats for the last 30 days
+  const profileStats = await getProfilesStats(
+    new Date(new Date().setDate(new Date().getDate() - 30)),
+  );
+  const oneRepActivations = profileStats?.total_active || 0;
+  const scanLimit = oneRepActivations > monthlySubscriberQuota;
```

Comment:
Should this be `scanLimit` or something more&hellip; boolean-y?
Like `atScanLimit` or `scanLimitReached`?

Reply:
thanks fixed